tests/unit/cndb/cndb_test.c

```diff
@@ -373,12 +375,25 @@ create_kvset(
     ASSERT_EQ(0, err);
 }
 
+uint64_t tgt_nodeid = 20;
 uint g_cb_ctr;
 
+static merr_t
+replay_full_move_cb(void *ctx, struct kvset_meta *km, u64 kvsetid)
```

Comment:
uint64_t please

Reply:
Same here, but I do believe that any identifier that ends with "_t" is reserved for use by the implementation.  I really hate the committee for carving that out.  Oh, and FWIW, there are many other similar reservations on identifiers (like ending with "_MAX" for another, screw them...)

Reply:
Case in point:  Veritas needed to grow their flags field they used in a zillion places from 32 to 64 bits.  Being defined as a uint32_t made that effort quite the nightmare...

Reply:
I "_t" is reserved for POSIX, but damn is it useful to identify a typedef when reading code.